File: week-3/week3-kalpanadhaker/q3.c
#include <stdio.h>
int main()
{
    int n, r, sum = 0;
    printf("Enter the number :");
    scanf("%d", &n);
    while (n > 0)
    {
        r = n % 10;
        sum = sum+ r; 
        n = n / 10;
    }
printf("Sum : %d", sum);
    return 0;
}

File: week-4/week4-DevChhiroliya/q3.py
lower_bounding=int(input("Enter the lower bounding number:"))
upper_bounding=int(input("Enter the upper bounding number:"))
print("Prime numbers between",lower_bounding,"and",upper_bounding,"are:")
for i in range(lower_bounding,upper_bounding+1):
    if i>1:
        for j in range(2,i):
            if i%j==0:

                break
        else:
            print(i)